amqp/transport.py

```diff
@@ -343,7 +343,7 @@ def _wrap_socket_sni(self, sock, keyfile=None, certfile=None,
                          ciphers=None, ssl_version=None):
```

Comment:
Do we still need the arguments we removed from the [opts ](https://github.com/celery/py-amqp/pull/327/files#diff-e3b09014ce12269dff5eece1f9f7c082L361-L371) dictionary?
Some of them are unused and I think they either should be used or removed from the signature.

This is a private method so we can change its signature at will.

Reply:
Done, we can go ahead and drop them